README.md

```diff
@@ -162,14 +163,24 @@ abstraction they approach. For most users, the right entry point is:
   to help create complex Kubernetes objects out of smaller objects
 
 `k.libsonnet` is built on top of several lower-level utility libraries
-that extends the object model and functions of `Jsonnet` to implement
-the Kubernetes API. These are generated directly from the swagger
-spec, and are organized by the API groups they belong to:
+that extends the object model and functions of **Jsonnet** to implement
+the Kubernetes API. These are generated directly from the Swagger
+definition, and are organized by the API groups they belong to:
 
 * `ksonnet.beta.1/apps.v1beta1.libsonnet`
 * `ksonnet.beta.1/core.v1.libsonnet`
 * `ksonnet.beta.1/extensions.v1beta1.libsonnet`
 
+## Mixins
+
+Mixins are a core feature of **Ksonnet**. Conceptually, they provide dynamic inheritance, at 
+runtime instead of compile time, which lets you combine them freely to modify objects or 
+create new ones.
+
+**Ksonnet** ships with a large library of built-in mixins, or you can write your own custom mixins. 
+The [tutorial](/docs/TUTORIAL.md) shows you how to create a custom mixin that you can then 
```

Comment:
I bet you've already thought of this, so just for my own education: this is meant to work on the website, right?